api-spec/transactions-api.yaml

```diff
@@ -392,6 +392,9 @@ components:
             mapping:
               postepay: "#/components/schemas/PostePayAuthRequestDetails"
               card: "#/components/schemas/CardAuthRequestDetails"
+        threeDsData:
```

Comment:
It should be a `CardAuthRequestDetails` field

Reply:
Are you sure about this point? From my point of view 3dsdata field is not a property of the _creditcard_ object. Thinking when we will manage a wallet, I don't think that the 3dsdata will be part of the credit card, while it will be for sure part of the payment authorization step.